subprojects/smoke-test/smoke-test.gradle.kts

```diff
@@ -49,6 +50,7 @@ dependencies {
     smokeTestImplementation(project(":coreApi"))
     smokeTestImplementation(project(":testKit"))
     smokeTestImplementation(project(":internalIntegTesting"))
+    smokeTestImplementation(project(":launcher"))
```

Comment:
Why is `:launcher` needed?

Reply:
`:launcher` is needed for using the `DaemonLogsAnalyzer`, since `DaemonRegistry` lives in `:launcher`.